lib/plugins/aws/package/compile/events/httpApi.js

```diff
@@ -274,15 +274,39 @@ Object.defineProperties(
       const userAuthorizers = userConfig.authorizers;
       const authorizers = (this.config.authorizers = new Map());
       if (userAuthorizers) {
+        if (typeof userAuthorizers !== 'object') {
+          throw new this.serverless.classes.Error(
+            'serverless.yaml is malformed at provider httpApi authorizers - should be an object with names as keys',
+            'MALFORMED_YAML'
+          );
+        }
         if (userConfig.id) {
           throw new this.serverless.classes.Error(
             'Cannot setup authorizers for externally configured HTTP API',
             'EXTERNAL_HTTP_API_AUTHORIZERS_CONFIG'
           );
         }
         for (const [name, authorizerConfig] of Object.entries(userAuthorizers)) {
+          // todo uncomment code when authorizer type is supported
+
+          // if (!authorizerConfig.type) {
```

Comment:
This check, whenever we will have support for types, should be a part of schema as in other cases instead of being an explicit check here